docs/docs/policies/actions.md

```diff
@@ -0,0 +1,40 @@
+# Actions
+
+Policies support global actions and specific actions per event. The supported actions now are:
+
+- log
+- webhook
+
+## Default Action
+
+The default action is the action executed for every event in the policy:
```

Comment:
Could we rewrite this sentence as:
"Every policy has a default action that will be executed for every event in same the policy"